crypt-bcrypt.c

```diff
@@ -870,20 +870,21 @@ BF_crypt (const char *key, const char *setting, unsigned char *output,
  * setting.
  */
 void
-crypt_bcrypt_rn (const char *key, const char *setting,
-                 uint8_t *output, size_t o_size,
-                 void *scratch, size_t s_size)
+crypt_bcrypt_rn (const char *phrase, size_t ARG_UNUSED (phr_size),
+                 const char *setting, size_t ARG_UNUSED (set_size),
+                 uint8_t *output, size_t out_size,
+                 void *scratch, size_t scr_size)
 {
   /* This shouldn't ever happen, but...  */
-  if (o_size < BF_HASH_LENGTH || s_size < sizeof (struct BF_buffer))
+  if (out_size < BF_HASH_LENGTH || scr_size < sizeof (struct BF_buffer))
```

Comment:
`set_size` must match `BF_SETTING_LENGTH` as well…

Reply:
Same deal here, BF_SETTING_LENGTH will be shorter than set_size if setting is a previously generated hash.

Reply:
Allrighty, my bad.